grpc/src/main/java/com/linecorp/armeria/internal/grpc/GrpcRequestUtil.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.internal.grpc;
+
+import javax.annotation.Nullable;
+
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+/**
+ * Utilities for introspecting a GRPC request.
+ */
+public final class GrpcRequestUtil {
```

Comment:
You're right - at first glance I thought it'd be a generic utility I'd need when writing the client, but a client never has to introspect the request since it makes it!

Fixed